CIME/bless_test_results.py

```diff
@@ -180,6 +180,16 @@ def bless_history(test_name, case, baseline_name, baseline_root, report_only, fo
             if not report_only and (
                 force or input("Update this diff (y/n)? ").upper() in ["Y", "YES"]
             ):
+                # Sometimes, in order to get things passing, files have to be removed
+                # from the baseline area.
+                for line in cmp_comments.splitlines():
+                    if NO_ORIGINAL in line:
+                        file_to_remove = (
+                            line.split(NO_ORIGINAL)[0].split()[-1].strip("'")
+                        )
+                        logger.info(f"Remove stale baseline file {file_to_remove}")
```

Comment:
You can't use f style formatting here.

Reply:
Why not?

Reply:
Oh, looks like the CI version of pylint doesn't like it. The one I used had no issue with it, but I can change it.

Reply:
Is this an optimization issue?
https://docs.python.org/3/howto/logging.html#optimization